packages/main/src/themes/sap_horizon_hcw/Token-parameters.css

```diff
@@ -8,12 +8,12 @@
 	--ui5_token_focus_pseudo_element_content: '';
```

Comment:
--_ui5_token_selected_hover_background: var(--sapButton_Hover_Background); -> Could you please check whether that is the expected value, since in the specification it is mentioned background: --sapButton_Selected_Hover_Background  and there is no delta specified for the HC themes.